tests/components/modbus/test_modbus_light.py

```diff
@@ -1,31 +1,42 @@
 """The tests for the Modbus light component."""
```

Comment:
Please rename this module to `test_light.py`.

Reply:
Done. Will rename all test* later.